include/zephyr/net/net_pkt.h

```diff
@@ -509,7 +509,7 @@ static inline void net_pkt_set_eof(struct net_pkt *pkt, bool eof)
 #if defined(CONFIG_NET_ROUTE)
```

Comment:
The commit subject `net: Code improvement` is too generic. We could change that to
`net: Follow coding style when converting int to bool`

Reply:
Sorry for that, I changed comment as you suggest.